collection-scripts/gather

```diff
@@ -84,6 +112,16 @@ else
   find /must-gather/clusters/*/namespaces/*/pods/ -name '*.log' -delete
 fi
 
+# Wipe secrets from migrated-namespaces data
+echo "Scrubbing secrets collected from migrated namespaces"
+find must-gather/clusters/*/migrated-namespaces/namespaces/*/*/ -name *secrets.yaml* -delete
+
+# Shorten logs from migrated-namespaces to last 100 lines
+echo "Shortening logs collected from migrated namespaces to last 100 lines"
+for logfile in $(find must-gather/clusters/*/migrated-namespaces/namespaces/*/*/ -name *.log); do
+  tail -100 "${logfile}" > tmp.log && mv tmp.log "${logfile}"
```

Comment:
I am not sure I understand where these logs are coming from. Are these logs from the Rsync/Stunnel pods?

Reply:
This is any cluster logs gathered from the `migrated-namespaces` (e.g. the namespaces that were on `migplan.spec.namespaces`), so this is Pod logs for Rsync, Stunnel, Stage Pods, _but also_ any other apps running in those namespaces.

The amount of logs that could be in these namespaces is potentially unbounded since this is where user apps run, so I thought it would make sense to only capture the last bits which would probably contain errors for Rsync or Stunnel or whatever. Do you think 100 lines is enough? 

I also considered if we should _only_ keep the logs for Rsync, Stunnel, Stage Pods, but since `oc adm inspect` grabs everything I'd need to know the naming conventions (like a regex) for all of those Pod names so I could delete everything except those. I like this idea better but it may be more brittle since we're still moving containers around (e.g. combining Rsync and Stunnel)

Reply:
@djwhatle Agreed. I am unsure about gathering user application logs though. Are we supposed to be doing that at all?

Reply:
There's not a great reason for us to. I think you make a good point, I should refine this further so that it only gathers logs for Pods belonging to us.